examples/plugin/plugin.toml

```diff
@@ -9,7 +9,7 @@ address = "0.0.0.0:19000"
 userprovidersvc = "localhost:19000"
 
 [grpc.services.userprovider]
-driver = "./json"
+driver = "https://github.com/jimil749/json"
```

Comment:
Also, just to demo the multiple ways of adding plugins, let's have one local and one hosted remotely

Reply:
Sure, I'll do that.